packages/desktop-client/src/components/reports/GraphButton.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { forwardRef, type HTMLProps } from 'react';
+
+import { type CSSProperties, theme } from '../../style';
+import Button from '../common/Button';
+import HoverTarget from '../common/HoverTarget';
+import Text from '../common/Text';
+import { Tooltip } from '../tooltips';
+
+type GraphButtonProps = HTMLProps<HTMLButtonElement> & {
+  selected?: boolean;
+  style?: CSSProperties;
+  onSelect;
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏missing type definition